e2e-tests/.gitignore

```diff
@@ -55,3 +55,11 @@ site
 # Cypress
```

Comment:
The following lines are not required anymore, right? 

# Cypress
**/cypress/downloads
**/cypress/screenshots